elasticsearch/run.sh

```diff
@@ -60,4 +57,18 @@ else
 	exit 1
 fi
 
-exec /usr/share/elasticsearch/bin/elasticsearch
+TIMES=60
```

Comment:
I was thinking the background process would kill ES by exiting 255, but
that was just my poor understanding of bash. Five minutes of trying should
be fine, LGTM :)

On Tue, Sep 13, 2016 at 4:38 PM, Eric Wolinetz <notifications@github.com>
wrote:

> In elasticsearch/run.sh
> <https://github.com/openshift/origin-aggregated-logging/pull/219#discussion_r78639511>
> :
>
> > @@ -60,4 +57,18 @@ else
> >  	exit 1
> >  fi
> >
> > -exec /usr/share/elasticsearch/bin/elasticsearch
> > +TIMES=60
>
> @sosiouxme <https://github.com/sosiouxme> i removed the exit 255 since it
> probably isn't doing what i think/want it to do.
> I also bumped us up to 300 from 60; I considered a do-while but I didn't
> want this out these spinning forever if someone ES gets stuck...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/219/files/2ece1d7d040c082b95982d655568b2781862e542#r78639511>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABz-vgVvGaeTGvO2FnJZZq3-QKVgjUzks5qpwnhgaJpZM4Jz_p->
> .
>